dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -83,11 +91,6 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         GetWaitHandleForArgs().create();
```

Comment:
Race condition here. This needs to be done before calling into CoRegisterClassObject. Otherwise we can get an incoming payload before we create the event.